src/components/TaskList/TaskList.tsx

```diff
@@ -20,15 +21,7 @@ function TaskList() {
     <ul className="task-list">
       {tasks.map((task) => {
         return (
-          <li
-            key={task.id}
-            className={task.complete ? "task complete" : "task"}
-          >
-            <span className="title">{task.title}</span>
-            <span>
-              <ListCheck />
-            </span>
-          </li>
+          <Task id={task.id} complete={task.complete} title={task.title} />
```

Comment:
`key` prop is missing.
Since Task component props and value of `task` within map loop are same, you can use spread syntax for clean code.
Eg,
`tasks.map(props => <Task {...props} />`